test/controllers/reports_controller_test.rb

```diff
@@ -63,10 +63,12 @@ def test_new_report_with_incomplete_details
       category = "other"
       post reports_path(:report => {
                           :category => category,
-                          :issue => { :reportable_id => 1, :reportable_type => "User" }
+                          :issue => { :reportable_id => target_user.id, :reportable_type => "User" }
```

Comment:
For some reason swapping to `render` instead of redirect causes our test user to get created (or _more often_ get created) with some id other than 1
but anyway it should probably have always been `target_user.id` to make it less brittle.